.github/workflows/ClangFormat.yml

```diff
@@ -16,4 +16,4 @@ jobs:
       - name: "Find source files"
         run: find . -name "*.cpp" -or -name "*.hpp" > sourceFiles.txt
       - name: "Check format"
-        run: clang-format-16 --Werror --dry-run --style="file:.clang-format" --verbose --files=sourceFiles.txt
\ No newline at end of file
+        run: clang-format-17 --Werror --dry-run --style="file:.clang-format" --verbose --files=sourceFiles.txt
```

Comment:
Since you touch this line anyway, would it be possible to use the respective `format-dry-run` make target?

Reply:
See my review message for additional details.

Reply:
Changed to format-dry-run